### ISSUE TITLE
Connecting to Server

### ISSUE THREAD

**Issue by felix28** [Johni0702/mumble-web#17]
Hi,
I am using LAMP in Ubuntu 16.04 with nodeJS V6 and npm installed.
I already clone the repository then run npm build so I have now a dist folder.
My website is also using HTTPS. This is the link: https://ts.config.ecoachmanager.com/

I thought I can now use this mumble web after I make my site into HTTPS.

Please help me.
How I can connect to server?

By the way I am not knowledgeable in websockets.

**Comment by Johni0702** [comment]
mumble-web can only create websocket connections but any ordinary mumble server will only accept tcp connections, therefore you need to setup a websocket to tcp proxy [as described in the README](https://github.com/Johni0702/mumble-web#setup).

**Comment by felix28** [comment]
Do I really need to make my website into HTTPS?

Last 2 days, I made it work without HTTPS. But today, it is not working.

**Comment by Johni0702** [comment]
Yes, TLS usage is currently [hard-coded](https://github.com/Johni0702/mumble-web/blob/906ef4e0276ef8d8866071cac11be1f9d2131dd1/app/index.js#L186) and there is really no good reason not to use it anyway.
